docs/Installing.asciidoc

```diff
@@ -624,6 +628,61 @@ This will allow the workers to download the assets from the webUI and use them
+unit tests have a look at the <<Contributing.asciidoc#contributing,developer guide>>. For a concrete
+example some developers use under openSUSE Tumbleweed have a look at the
+https://github.com/Martchus/openQA-helper[openQA-helper repository].
+
+=== Repository URLs
+* os-autoinst: https://github.com/os-autoinst/os-autoinst
+    - the "backend" (thing that executes tests and starts/controls the SUT e.g. using QEMU)
+* openQA: https://github.com/os-autoinst/openQA
+    - mainly the web UI and accompanying daemons like the scheduler
+    - the worker (thing that starts the backend and uploads results to the web UI)
+    - documentation
+* test distribution: e.g. https://github.com/os-autoinst/os-autoinst-distri-opensuse for openSUSE
+    - the actual tests
+* needles: e.g. https://github.com/os-autoinst/os-autoinst-needles-opensuse for openSUSE
+    - reference images if not already included in the test distribution
+
+=== Dependencies
+Have a look at the packaged version (e.g. +openQA.spec+ within the root of the openQA repository)
+for all required dependencies. For development build time dependencies need to be installed as well.
+Recommended dependencies such logrotate can be ignored.
+
+=== Starting daemons
+Although it likely makes not much sense to use the systemd unit files directly it still makes
+sense to have a look at them to see how the different daemons are started. They are found in
+the +systemd+ directory of the openQA repository. You can substitue +/usr/share/openqa/+ with the
+path of your openQA Git checkout.
+
+Of course you can ignore the user specified in these unit files and instead start everything as your
+regular user. However, you need to ensure that your user has the permission to the "openQA base directory".
+That is not the case by default so it makes sense to <<Contributing.asciidoc#_customize_base_directory,customize it>>.
+
+Note that the web UI daemon will pull required JavaScript/CSS libraries automatically when started
+the first time. This might take a while and requires an internet connection.
+
+You do *not* need to setup an additional web server because the daemons already provide one. The port
+under which a service is available is logged on startup (the main web UI port is 9625 by default). Local
+workers need to be configured to connect to the main web UI port (add +HOST = http://localhost:9526+ to
++workers.ini+).
+
+For an overview over all daemons/processes/ports and their connections have a look at
+the link:images/architecture.svg[architecture diagram].
+
+=== Setting up the database
+One also needs to setup a PostgreSQL database for openQA manually owned by your regular user.
+The steps to do that are documented in the <<Contributing.asciidoc#setup-postgresql,developer guide>>.
+Note that an empty database is sufficient to get started (openQA will default-initialize it on the first
+startup). So you can skip the steps for importing an existing database dump.
+
```

Comment:
I didn't check in detail right now but I am pretty sure these parts are already described elsewhere

Reply:
I sometimes mentioned it e.g. in the chat when pointing people to that section of the developer guide. But I guess it makes more sense make that section itself more general. (When I wrote it initially devs used sqlite and setting up PostgreSQL was rally only relevant for importing production data.)

Reply:
hm, ok. Installing.asciidoc only shortly describes the necessary configuration for a database.